### PR TITLE
HW-54158: Fix code comment and readme.md for listPrepaidCardReceipts

### PR DESCRIPTION
**Pull request by oleksii-lubianyi-epam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -344,6 +344,29 @@ Hyperwallet.shared.listPrepaidCards(queryParam: prepaidCardQueryParam) { (result
 }
 ```
 
+### List Prepaid Card Receipts
+```swift
+let receiptQueryParam = HyperwalletReceiptQueryParam()
+receiptQueryParam.createdAfter = ISO8601DateFormatter.ignoreTimeZone.date(from: "2016-12-01T00:00:00")
+receiptQueryParam.createdBefore = ISO8601DateFormatter.ignoreTimeZone.date(from: "2020-12-31T00:00:00")
```
Comment (jsaini-hw) [inline README.md]: No need of setting createdBefore so you can remove this line

File: Sources/Hyperwallet.swift [hunk 2]
```diff
@@ -573,8 +573,6 @@ public final class Hyperwallet {
     /// `HyperwalletReceiptQueryParam` object, if it is not nil. Otherwise the default ordering and
     /// filtering will be applied.
     ///
-    /// * Offset: 0
```
Comment (jsaini-hw) [inline Sources/Hyperwallet.swift]: Update the comment to:
' /// Returns the list of receipts for the User associated with the Prepaid card token.
   ///
   /// The filtering of `HyperwalletReceipt` will be based on the criteria specified within the
   /// `HyperwalletReceiptQueryParam` object. CreatedAfter needs to be provided to get the receipts.
  /// Receipts are returned sorted in ascending order of creation date
   ///
   /// * Created Before: N/A
   /// * Created After: “Some Date”
   /// * Currency: All '

File: README.md (hunk 1 above)
Comment (jsaini-hw) [reply README.md]: Also add List User Receipts

Comment (oleksii-lubianyi-epam) [reply README.md]: Removed unnecessary line. Added description for List User Receipts

File: Sources/Hyperwallet.swift (hunk 2 above)
Comment (oleksii-lubianyi-epam) [reply Sources/Hyperwallet.swift]: Updated

**Pull request merged**